Src/Witsml/CommonConstants.cs

```diff
@@ -7,5 +7,35 @@ public static class CommonConstants
 {
     public const int DefaultNumberOfRoundedPlaces = 3;
     public const string DataSeparator = ",";
+    public const string PropertySeparator = ".";
     public const string NewLine = "\n";
+    public const int DefaultClientRequestTimeOut = 90;
+    public const int DefaultReloadInterval = 15;
```

Comment:
Add please unit of the value into name, like DefaultClientRequestTimeOutSecs, DefaultReloadIntervalMins with comment // in seconds, // in minutes.....